examples/hotrod/cmd/flags.go

```diff
@@ -38,11 +38,8 @@ var (
 	jaegerUI string
 )
 
-const expvarDepr = "(deprecated, will be removed after 2024-01-01 or in release v1.53.0, whichever is later) "
-
 // used by root command
 func addFlags(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVarP(&metricsBackend, "metrics", "m", "prometheus", expvarDepr+"Metrics backend (expvar|prometheus). ")
```

Comment:
I suggest we keep this with just `prometheus` as the only allowed value. In the future we might support otlp (although more likely it will come in the form of OTEL config)